packages/loopring_v3/contracts/impl/libexchange/ExchangeGenesis.sol

```diff
@@ -39,7 +39,7 @@ library ExchangeGenesis
         S.exchangeCreationTimestamp = now;
         S.loopring = ILoopringV3(_loopringAddress);
         S.operator = _operator;
-        S.onchainDataAvailability = _onchainDataAvailability;
+        S.rollupEnabled = _rollupEnabled;
```

Comment:
Add `maxAgeDepositUntilWithdrawable = MAX_AGE_DEPOSIT_UNTIL_WITHDRAWABLE_UPPERBOUND();`, otherwise the default value is 0.

Reply:
done